File: javascript/combustivel.js
let a = 0, g = 0, d = 0, codigo;

do {
    codigo = parseInt(prompt("Informe um codigo (1, 2, 3) ou 4 para parar"));
    
    if (codigo == 1) {
        a++;
    } else if (codigo == 2) {
        g++;
    } else if (codigo == 3) {
        d++;
    }
} while (codigo != 4);

console.log("MUITO OBRIGADO");
console.log("Alcool = " + a);
console.log("Gasolina = " + g);
console.log("Diesel = " + d);

File: python/combustivel.py
qtd_alcool = 0
qtd_gasolina = 0
qtd_diesel = 0

codigo = int(input("Informe um codigo (1, 2, 3) ou 4 para parar: "))

while codigo != 4:
    if codigo == 1:
        qtd_alcool += 1
    elif codigo == 2:
        qtd_gasolina += 1
    elif codigo == 3:
        qtd_diesel += 1

    codigo = int(input("Informe um codigo (1, 2, 3) ou 4 para parar: "))

print("MUITO OBRIGADO")
print("Alcool:", qtd_alcool)
print("Gasolina:", qtd_gasolina)
print("Diesel:", qtd_diesel)
